.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+ECJ_VERSION="4.6.1"
```

Comment:
@romani  - are you sure you want `staging` release? I've found on the same server `release` folder - https://repo.eclipse.org/content/repositories/eclipse-releases/org/eclipse/jdt/org.eclipse.jdt.core/
Should we use latest from here?